sendgrid.cfc

```diff
@@ -975,7 +2289,8 @@ component output="false" displayname="SendGrid.cfc"  {
         'path' : fullApiPath,
         'params' : serializeJSON( queryParams ),
         'response' : apiResponse.fileContent,
-        'responseHeaders' : apiResponse.responseheader
+        'responseHeaders' : apiResponse.responseheader,
+        'arguments' : arguments
```

Comment:
Is there something in particular in the arguments that you found helpful to include here? In other wrappers I've explicitly included the body payload. Just wondering, if something is missing, if we should include it more specifically than `arguments`.

Reply:
Yes.  When debugging it was helpful full to see all the arguments which were passed in.   However, once I completed running through the tests it was not needed.   It was helpful to debug POST methods when complex body structures were passed in.

I will remove it and push update.